guide/blueprints/custom-entities.md

```diff
@@ -262,6 +261,30 @@ so that the `$message` we passed above gets logged and reported back:
           period: 1s
           command: tail -1 server-input
 
+#### ContainerSensor
+
+This blueprint uses initializers to define a `ContainerSensor` on a `BasicStartable` entity so that a random number is generated by executing a command on a container for this specific purpose, and destroyed afterwards.
+
+{% highlight yaml %}
+
+name: entity-with-container-sensor
+services:
+- type: 'org.apache.brooklyn.entity.stock.BasicStartable:1.1.0-SNAPSHOT'
+  brooklyn.initializers:
+  - type: org.apache.brooklyn.tasks.kubectl.ContainerSensor
+    brooklyn.config:
+    image: perl
+    imagePullPolicy: never
```

Comment:
`imagePullPolicy` value is probably case-sensitive.

Reply:
@algairim  Nope it is not. I added an enum for it.